.env

```diff
@@ -0,0 +1,6 @@
+VITE_FIREBASE_API_KEY="AIzaSyAe9rC1nZrbi03G4LVSEr4chHBIOUyP5eU"
```

Comment:
why this env file is in your PR ?

Reply:
mam you told that create read about env file and use as fire base config

Reply:
this file should not be in you PR , write this in gitignore